packages/combo-box/test/typings/combo-box.types.ts

```diff
@@ -69,41 +81,98 @@ narrowedComboBox.addEventListener('selected-item-changed', (event) => {
-assertType<ThemableMixinClass>(narrowedComboBoxLightElement);
+/* ComboBoxLight */
+const genericComboBoxLight = document.createElement('vaadin-combo-box-light');
+assertType<ComboBoxLight>(genericComboBoxLight);
 
-narrowedComboBoxLightElement.addEventListener('custom-value-set', (event) => {
+const narrowedComboBoxLight = genericComboBoxLight as ComboBoxLight<TestComboBoxItem>;
+assertType<ComboBoxDataProviderMixinClass<TestComboBoxItem>>(narrowedComboBoxLight);
+assertType<ComboBoxMixinClass<TestComboBoxItem>>(narrowedComboBoxLight);
+assertType<ThemableMixinClass>(narrowedComboBoxLight);
+
+narrowedComboBoxLight.addEventListener('custom-value-set', (event) => {
   assertType<ComboBoxLightCustomValueSetEvent>(event);
   assertType<string>(event.detail);
 });
 
-narrowedComboBoxLightElement.addEventListener('opened-changed', (event) => {
+narrowedComboBoxLight.addEventListener('opened-changed', (event) => {
   assertType<ComboBoxLightOpenedChangedEvent>(event);
   assertType<boolean>(event.detail.value);
 });
 
-narrowedComboBoxLightElement.addEventListener('invalid-changed', (event) => {
+narrowedComboBoxLight.addEventListener('invalid-changed', (event) => {
   assertType<ComboBoxLightInvalidChangedEvent>(event);
   assertType<boolean>(event.detail.value);
 });
 
-narrowedComboBoxLightElement.addEventListener('value-changed', (event) => {
+narrowedComboBoxLight.addEventListener('value-changed', (event) => {
   assertType<ComboBoxLightValueChangedEvent>(event);
   assertType<string>(event.detail.value);
 });
 
-narrowedComboBoxLightElement.addEventListener('filter-changed', (event) => {
+narrowedComboBoxLight.addEventListener('filter-changed', (event) => {
   assertType<ComboBoxLightFilterChangedEvent>(event);
   assertType<string>(event.detail.value);
 });
 
-narrowedComboBoxLightElement.addEventListener('selected-item-changed', (event) => {
+narrowedComboBoxLight.addEventListener('selected-item-changed', (event) => {
   assertType<ComboBoxLightSelectedItemChangedEvent<TestComboBoxItem>>(event);
   assertType<TestComboBoxItem | null | undefined>(event.detail.value);
 });
+
+// ComboBoxLight properties
+assertType<() => boolean>(narrowedComboBoxLight.checkValidity);
+assertType<() => boolean>(narrowedComboBoxLight.validate);
+assertType<() => void>(narrowedComboBoxLight.close);
+assertType<() => void>(narrowedComboBoxLight.open);
+assertType<boolean | null | undefined>(narrowedComboBoxLight.autoOpenDisabled);
+assertType<boolean | null | undefined>(narrowedComboBoxLight.opened);
+assertType<boolean>(narrowedComboBoxLight.invalid);
+assertType<boolean>(narrowedComboBoxLight.disabled);
+assertType<boolean>(narrowedComboBoxLight.readonly);
+assertType<string>(narrowedComboBoxLight.value);
+
+// ComboBoxLight mixins
+assertType<ElementMixinClass>(narrowedComboBoxLight);
+assertType<DisabledMixinClass>(narrowedComboBoxLight);
+assertType<InputConstraintsMixinClass>(narrowedComboBoxLight);
```

Comment:
Hmm, couldn't find the combo-box-light extending `InputConstraintsMixinClass`.

Reply:
This class comes with `PatternMixin`.

Reply:
Removed. It was passing because `InputConstraintsMixinClass` has only `checkValidity` method.